cmd/apiserver/server.go

```diff
@@ -60,12 +57,87 @@ func (s *server) createCalculationBulk(c *gin.Context) {
 	}
 
 	if err := s.client.Create(s.ctx, bulk); err != nil {
-		responseError(c, "couldn't create calculation", err)
+		responseError(c, "couldn't create calculation bulk", err)
 	} else {
 		c.JSON(http.StatusOK, gin.H{"data": bulk})
 	}
 }
 
+func (s *server) createWorkerPool(c *gin.Context) {
+	body, err := ioutil.ReadAll(c.Request.Body)
+	if err != nil {
+		responseError(c, "couldn't read body", err)
+		return
+	}
+
+	workerPool := &workersv1.WorkerPool{}
+
+	var NameBody struct {
```

Comment:
This will require the user to send a JSON request as `{ "name": "test"}`. We just need to read a param from the body. Instead of forcing the user to send a JSON content type, he can just send a `application/x-www-form-urlencoded` content-type with that param included.